openrouteservice/src/main/java/org/heigit/ors/api/controllers/MatrixAPI.java

```diff
@@ -111,6 +115,8 @@ public ResponseEntity<Object> handleReadingBodyException(final Exception e) {
             return errorHandler.handleUnknownParameterException(new UnknownParameterException(MatrixErrorCodes.UNKNOWN_PARAMETER, ((UnrecognizedPropertyException) cause).getPropertyName()));
         } else if (cause instanceof InvalidFormatException) {
             return errorHandler.handleStatusCodeException(new ParameterValueException(MatrixErrorCodes.INVALID_PARAMETER_FORMAT, ((InvalidFormatException) cause).getValue().toString()));
+        } else if (cause instanceof ConversionFailedException) {
+            return errorHandler.handleStatusCodeException(new ParameterValueException(MatrixErrorCodes.INVALID_PARAMETER_VALUE, Objects.requireNonNull(((ConversionFailedException) cause).getValue()).toString()));
```

Comment:
Objects.requireNonNull is not needed here.
```suggestion
return errorHandler.handleStatusCodeException(new ParameterValueException(MatrixErrorCodes.INVALID_PARAMETER_VALUE, ((ConversionFailedException) cause).getValue().toString()));
```